Gruntfile.js

```diff
@@ -15,23 +15,49 @@ module.exports = function(grunt) {
         serverToken: '<%= secrets.serverToken %>'
       },
       email: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Yo',
         src: ['test/email.html']
       },
       bulk: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Hey',
         src: ['test/*.html']
       }
-    }
+    },
+
+    "postmark-templates": {
+      options: {
+        serverToken: '<%= secrets.serverToken %>',
+      },
+      build: {
+        name: "testing-template-node-js-" + new Date().valueOf(),
+        subject: "Testing grunt-postmark-templates",
+        // NOTE these are read from filesystem. globbing not supported
```

Comment:
I know that grunt has fancy _src_ and _dest_ file handling but in my first pass I did it in a less idiomatic way.